translations/monero-core_pt-br.ts

```diff
@@ -1037,37 +1037,37 @@
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="60"/>
         <source>GUI version: </source>
-        <translation type="unfinished"></translation>
+        <translation>Versão da Interface</translation>
```

Comment:
missing `:` + whistespace

Reply:
Thank you. If @vp1111 could review the file today, I send the PR again tonight (about 10:00pm UTC).

Reply:
No need to send another PR, you can just push the new commits on this one